cypress/integration/functional/balances.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { AddressToken } from '@defichain/whale-api-client/dist/api/address'
+
+const tokens: AddressToken[] = [
+  {
+    id: '0',
+    symbol: 'DFI',
+    amount: '50',
+    name: 'Defi',
+    isDAT: true,
+    isLPS: false,
+    symbolKey: 'DFI'
+  },
+  {
+    id: '1',
+    symbol: 'tBTC',
+    amount: '100',
+    name: 'Playground BTC',
+    isDAT: true,
+    isLPS: false,
+    symbolKey: 'tBTC'
+  }
+]
```

Comment:
don't write a test like this, stub based testing is harder to maintain in the long run use a more functional approach where you actually write out the test contract instead of looping them through.

A test should be more explicit with fewer references and inferences. It should be clearly explicit.

Reply:
I have removed this